PLEP-0002.md

```diff
@@ -0,0 +1,138 @@
+# PLEP-0002 -- Coordinating Committee
+
+| PLEP              | 2                                 |
+|-------------------|-----------------------------------|
+| title             | Coordinating Committee            |
+| author(s)         | Nick Murphy                       |
+| contact email     | namurphy@cfa.harvard.edu          |
+| date created      | 2017-11-15                        |
+| date last revised | 2017-12-01                        |
+| type              | process                           |
+| status            | in preparation                    |
+
+# Abstract
+
+PlasmaPy's **Coordinating Committee** (CC) shall oversee the PlasmaPy
+project and code development.
+
+# Detailed Description
+
+## Membership of Coordinating Committee
+
+### Limits on number of members
+
+The CC shall consist of at least 4 members and no more than 10
+members.  Within these limits, the CC may increase or decrease the
+number of directors serving on the CC, including for the purposes of
+staggering the terms of members.  The members of the Coordinating
+Committee shall be listed in [PLEP 
+3](https://github.com/PlasmaPy/PlasmaPy-PLEPs/blob/master/PLEP-0003.md).
+
+### Term of office
+
+All members shall be elected to serve terms of up to two years.
+However, the CC may extend the term until a successor has been
+elected.  A member may be re-elected at the end of their term.  The
+term of a CC member shall begin on January 1 of the first year in
```

Comment:
What about the current CC members? Are those terms going to be longer or shorter, or are we saying they begin officially on 01/01/18?